edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/version_2018_06_28/generatedCode/HttpWorkloadClient.cs

```diff
@@ -1,46 +1,125 @@
-// Copyright (c) Microsoft. All rights reserved.
 //----------------------
 // <auto-generated>
-//     Generated using the NSwag toolchain v11.17.6.0 (NJsonSchema v9.10.46.0 (Newtonsoft.Json v9.0.0.0)) (http://NSwag.org)
+//     Generated using the NSwag toolchain v13.3.0.0 (NJsonSchema v10.1.11.0 (Newtonsoft.Json v12.0.0.0)) (http://NSwag.org)
```

Comment:
This code was regenerated with nswag's dotnet 3 option

Reply:
Is there anything that we use from dotnet 3.1? If not I would say not to regenerate.